lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * This overlay is similar to gtfs-rt-vehicle-overlay in that it shows
+ * realtime positions of vehicles on a route using the otp-ui/transit-vehicle-overlay.
+ *
+ * However, this overlay differs in a few ways:
+ * 1) This overlay retrieves vehicle locations from OTP
+ * 2) This overlay renders vehicles as blobs rather than a custom shape
+ * 3) This overlay does not handle updating positions
+ * 4) This overlay does not render route paths
+ * 5) This overlay has a custom popup on vehicle hover
+ */
+import TransitVehicleOverlay from '@opentripplanner/transit-vehicle-overlay'
```

Comment:
Sort imports.

Reply:
Sorted manually in 9a7139548d78db23db07dfddf9940bd85078072b